### ISSUE TITLE
Fails to load propertly on cloud docker

### ISSUE THREAD

**Issue by GeoffreyEmery** [Zertz/tutum-blazing-wordpress#1]
Hello,
   Just throughing this into cloud docker this doesn't launch. I think there is is a issue in the port mapping from htproxy and varnish. Occasionaly it will load the varnish on 8080 but I was never able to get lb to connect to the varnish instance. .
  Docker:latest
  Deplpyment:DockerCloud:Azure.

**Comment by Zertz** [comment]
I haven't tinkered with this in a while, but I think it may be an issue with Tutum's haproxy image failing to link correctly.

**Comment by GeoffreyEmery** [comment]
yeah i agree that is that sounds like the issue. I can't seem to get a fix for this i have a a few modes here is the a current docker-compose.yml

`

db:
  image: mariadb
  environment:
    MYSQL_ROOT_PASSWORD: example



wordpress:
  image: wordpress
  links:
    - db:mysql
  ports:
    - 81:80


varnish:
  image: 'benhall/docker-varnish:latest'
  environment:
    - VARNISH_BACKEND_HOST=backend
    - VARNISH_BACKEND_PORT=80 
    - 'VIRTUAL_HOST=example.com,www.example.com'
  links:
    - wordpress:backend
  ports:
    - '8080:80'


lb:
  image: 'dockercloud/haproxy'
  links:
    - varnish
  ports:
    - '80:80'
  `

**Comment by Zertz** [comment]
Yeah, it has to do with the order in which the containers are loaded, sometimes haproxy would not pick up Varnish. I honestly don't use this particular stack anymore, but I'm guessing swapping out Tutum's image for the official haproxy image might do the trick.

**Comment by GeoffreyEmery** [comment]
Yeah i changed that out in that the docker-compose file its not doing anything, Do you have a better stack for a wordpress host that you use now?

**Comment by Zertz** [comment]
I do not self host WordPress anymore :100:

If you've already switched to the official haproxy image, you can also try using this [Varnish image](https://hub.docker.com/r/cloudposse/varnish/), which is far more recent and seems to be well maintained. @cloudposse
